code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -802,7 +802,6 @@
 /datum/dynamic_ruleset/roundstart/bloodsucker
 	name = "Bloodsuckers"
 	config_tag = "bloodsucker"
-	persistent = TRUE
```

Comment:
Ah, I didn't comment on the commit message here. This straight up does nothing. but make it (uselessly!) process with the default processing.

Reply:
Good to know, gamemodes have an awfully lot of ways that would seem to cause them to end on the antag death, so i wasnt really sure which one it was.